packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -81,10 +81,15 @@ export class AccountHandler {
   private async deleteAccountIfUnverified(request: AuthRequest, email: string) {
     try {
       const secondaryEmailRecord = await this.db.getSecondaryEmail(email);
-      // Currently, users can not create an account from a verified
+      // Currently, users cannot create an account from a verified
```

Comment:
I wish we could pull this out into a shared function `deleteAccountOrEmailIfUnverified`, but it seems there are sutble differences in the routes :/